yaptide/batch/string_templates.py

```diff
@@ -27,6 +28,9 @@
 cat << EOF > $DETECT_FILE
 {detect}
 EOF
+cat << EOF > $WATCHER_SCRIPT
+{watcher}
+EOF
```

Comment:
I am afraid using cat may string double qoutes and other unexpected consequences.
See for example:
https://unix.stackexchange.com/questions/532036/how-to-display-double-quotes

The same apply to other files.

Why not copy these files using fabric ?

Reply:
I wanted to minimize the number of commands running via ssh.
It worked for `.dat` files without any problems

Reply:
Yes, but what if we introduce `"` in the dat files ?
Also - some of the input files, like `sobp.dat` can be pretty large (~1MB). Putting that large files into strings is not a good idea.

What about on-the-fly compression of all what is needed to run SH12A into a single compressed file ? For example:
https://docs.python.org/3/library/zipfile.html

On the server side we can assume that `zip` is always available.